client/signup/config/flows.js

```diff
@@ -213,6 +213,15 @@ const flows = {
 		description: 'User-first signup flow',
 		lastModified: '2016-01-18',
 	},
+
+	wpcc: {
+		steps: [ 'oauth2-user' ],
+		destination: function( dependencies ) {
+			return dependencies.oauth2_redirect.split( '@' )[ 1 ];
```

Comment:
Do we need to worry about what happens if there isn't an `@` in the parameter? My feeling is that if that's the case then that means someone tampered with the parameter and it shouldn't work. Maybe we should still show an error?